src/views/Wallet/Settings/SettingsView.vue

```diff
@@ -48,7 +48,7 @@ export default {
   methods: {
     removeWallet () {
       this.$buefy.dialog.confirm({
-        message: this.$i18n.t('walletSettings.deleteWalletConfirm', { name: this.wallet.name }),
+        message: new Option(this.$i18n.t('walletSettings.deleteWalletConfirm', { name: this.wallet.name })).innerHTML,
```

Comment:
Oof I don't understand this one 🤔 can you explain what's happening here?

Reply:
Okay I understand it will escape the html. Well as translation labels are very likely to be adjusted by users it's save to use their html without consideration. And as the wallet.name is already validated before creation we can safely display the value being created. I'll revert this too 😇